tracer/test/Datadog.Trace.TestHelpers/AgentBehaviour.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="AgentBehaviour.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.TestHelpers
+{
+    public enum AgentBehaviour
+    {
+        /// <summary>
+        /// Normal agent behaviour
+        /// </summary>
+        NORMAL = 0,
```

Comment:
nit: repo style is to use normal case for these
```suggestion
        Normal = 0,
```